File: microservices-server-app/ProductOrderWebApi/Dto/UserInfoDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProductOrderWebApi.Dto
{
    public class UserInfoDto
    {
        public string Username { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Address { get; set; }
        public byte[] Picture { get; set; }
    }
}

File: my-react-app/src/models/UserInfoDto.js
class UserInfoDto {
    constructor(username, email, name, surname, address, picture) {
      this.username = username || "";
      this.email = email || "";
      this.name = name || "";
      this.surname = surname || "";
      this.address = address || "";
      this.picture = picture || null;
    }
  }
  
  export default UserInfoDto;
  